packages/backend/src/open_payments/payment/incoming_remote/service.test.ts

```diff
@@ -18,6 +18,7 @@ import {
 } from 'open-payments'
 import { GrantService } from '../../grant/service'
 import { RemoteIncomingPaymentError } from './errors'
+import { AccessToken } from 'open-payments/dist/types'
```

Comment:
```suggestion
import { AccessToken } from 'open-payments'
```

Reply:
`AccessToken` is currently not exported by `open-payments` but I could change that.

Reply:
I think would be better to export from `index` in `open-payments` 👍